documentation/src/docs/security/securing-distributed-mode-cluster.md

```diff
@@ -247,54 +63,31 @@ These steps are discussed in the following sub-sections.
 
 ### Configuring TLS and Auth on Server Side
 
-This step is about using the certificates, keys, keystores and truststores generated earlier to configure
-TLS and Auth for the services on the server side.
-
-Note that if you enable TLS and Auth on one service, you must enable them for all the other services too. Pravega
-does not support using TLS and Auth for only some instances of the services.
-
 You can configure the following services for TLS and Auth:
 
 1. Controller
 2. Segment Store
-3. Zookeeper (optionally)
-4. Bookkeeper (optionally)
+3. Zookeeper (optional)
+4. Bookkeeper (optional)
 
 For information about enabling TLS for Zookeeper and Bookeeper, refer to
 their documentation here:
 * [ZooKeeper SSL Guide](https://cwiki.apache.org/confluence/display/ZOOKEEPER/ZooKeeper+SSL+User+Guide)
 * [BookKeeper - Encryption and Authentication using TLS](https://bookkeeper.apache.org/docs/latest/security/tls/)
 
-Configuring security for Controllers and Segment Stores is discussed below.
-
 **Controller**
 
-Controller services can be configured in three different ways:
+Controller services can be configured in two different ways:
 
 1. By specifying the configuration parameter values directly in the `controller.config.properties` file. For example,
 
    ```
    controller.security.tls.enable=true
-   controller.security.tls.server.certificate.location=/etc/secrets/controller01.pem
-   ```
-2. By specifying configuration values via corresponding environment variables. For example,
-
-   ```bash
-   # TLS_ENABLED environment variable corresponds to Controller configuration parameter
-   # "controller.auth.tlsEnabled".
-   $ export TLS_ENABLED: "true"
-   ```
-
-   To identify the environment variables corresponding to the configuration parameter, inspect the default
-   `controller.config.properties` file and locate the variables.
-
-   ```
-   controller.security.tls.enable=${TLS_ENABLED}
-   controller.security.tls.server.certificate.location=${TLS_CERT_FILE}
+   controller.security.tls.server.certificate.location=/etc/secrets/controller01-server-cert.crt
```

Comment:
Use a filename here that matches the equivalent filename in `generating-tls-artifacts.md` for consistency.

Reply:
Done.